test/patterns/send_null/test_send_null.py

```diff
@@ -14,7 +14,7 @@ def test_one_send(self):
 
     def test_multi_level_invocation(self):
         lines = self.method_send_null_finder.value(Path(self.dir_path, 'Configuration.java'))
-        self.assertEqual(lines, [379, 442, 549, 638, 656, 830, 866, 1362, 2393, 2988, 3080, 3492, 3855])
+        self.assertEqual(lines, [379, 442, 549, 638, 656, 830, 866, 1362, 2393, 2874, 2988, 3080, 3492, 3758, 3855])
```

Comment:
Sounds like we found some new lines, that match pattern.

Reply:
we found new examples